pkgs/top-level/all-packages.nix

```diff
@@ -23285,10 +23285,12 @@ with pkgs;
   # Default libGLU
   libGLU = mesa_glu;
 
-  mesa = darwin.apple_sdk_11_0.callPackage ../development/libraries/mesa {
-    inherit (darwin.apple_sdk_11_0.frameworks) OpenGL;
```

Comment:
OpenGL no longer required?

Reply:
I don't think it ever was, actually - it's only ever dlopened from a fixed path.